PolicyServicePkg/Test/PolicyServicePkgHostTest.dsc

```diff
@@ -12,7 +12,7 @@
   PLATFORM_VERSION        = 0.1
   DSC_SPECIFICATION       = 0x00010005
   OUTPUT_DIRECTORY        = Build/PolicyServicePkg/HostTest
-  SUPPORTED_ARCHITECTURES = IA32|X64|AARCH64
+  SUPPORTED_ARCHITECTURES = IA32|X64

```

Comment:
@kuqin12 - For completeness, can you elaborate in the PR description what is not supported yet?

Reply:
Will do.

Reply:
Updated PR descriptions. We can work towards that if we want to start supporting ARM64 unit tests.